tests/data/duty.data.ts

```diff
@@ -0,0 +1,30 @@
+import { faker } from '@faker-js/faker';
+
+import { ObjectId } from 'mongodb';
+
+const createTestDuty = () => {
+  return {
+    name: faker.name.fullName(),
+    description: faker.lorem.sentence(),
+    location: faker.address.city(),
+    time: {
+      start: faker.date.future(1),
+      end: faker.date.future(1),
+    },
+    constraints: Array.from({ length: 3 }, (x, i) => {
+      return `constraint ${i}`;
+    }),
+    soldiersRequired: Math.round(Math.random() * 10),
+    value: Math.round(Math.random() * 10),
+    soldiers: [],
+  };
+};
+
+const createTestDutyWithId = () => {
```

Comment:
Why are you creating them with an id ??

Reply:
Because I need to know the id for the tests

Reply:
resolved